src/i18n/base.js

```diff
@@ -196,7 +196,10 @@ wb.i18nDict = {
 	"geo-allyttl": "@geo-allyttl@",
 	"geo-tgllyr": "@geo-tgllyr@",
 	"geo-hdnlyr": "@geo-hdnlyr@",
+
+	/* geo-bmapurl to be deprecated - do not use */
 	"geo-bmapurl": "@geo-bmapurl@",
+	"geo-bmap-url": "@geo-bmap-url@",
```

Comment:
@jvanulde this shouldn't affect compatibility because the i18n file is tied to a WET version. You shouldn't use the v4.0.12 i18n file with v4.0.24 for example.